contracts/implementations/streamr/StreamrManager.sol

```diff
@@ -0,0 +1,27 @@
+//SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.13;
+
+import "../../shared/Roles.sol";
+import "../../libraries/streamr/StreamrManagerStorage.sol";
+
+import "@solidstate/contracts/access/access_control/AccessControlInternal.sol";
+
+/**
+ * @title StreamrManager
+ * @notice Contract to manage information related to Streamr
+ */
+contract StreamrManager is AccessControlInternal {
```

Comment:
Any other name except manager :)

Reply:
Mapper?

Reply:
Let's give him a different name.

Reply:
What about `StreamrConfigurator`?

Reply:
lmaooo

Reply:
Oh you're being serious

Reply:
I genuinely thought it was a good idea 😶

Reply:
Well, no complaint from Yev. I think we are good now.